index.bs

```diff
@@ -1691,31 +1651,6 @@ which might be expressed as a bitmask in another language,
 use a dictionary object instead.
 This object can be passed around as easily as a single bitmask value.
 
-<h3 id="properties-vs-methods">Use properties and methods appropriately</h3>
```

Comment:
What happened to this section?

Reply:
Merged into #attributes-like-data along with #attributes-vs-methods, eliminating the redundancy of three sections describing appropriate behavior for attributes/properties vs. methods and lamenting their violation by `offsetTop`.

Reply:
I agree there is redundancy, and thank you for spotting it! However, right now it feels like a lot of the more concrete guidance was dropped in the process (e.g. the parts about what should equal what). Could you try to fold some of that in?

Reply:
I believe all the points are still there.
* "_Getters must not have any observable side effects._" is currently in all three sections and remains in the proposed merge.
* "_Getters should not perform any {complex,blocking,[inefficient]} operations._" is currently in all three sections and remains in the proposed merge.
* A variation of "_Ensure that `obj.attribute === obj.attribute` is always true. Don't create a new value each time the getter is called._" is currently in all three sections remains in the proposed merge with the least passive phrasing.
* "_[ensure that] given `obj.attribute = x`, [a subsequent] `obj.attribute === x` {is,should be} true._" is currently in all three sections and remains in the proposed merge. And I have now also incorporated the elaborations currently absent from [#attributes-like-data](https://w3ctag.github.io/design-principles/#attributes-like-data).
* Guidance that static objects should be new each time (and therefore returned from methods rather than attributes) is currently absent from [#attributes-like-data](https://w3ctag.github.io/design-principles/#attributes-like-data) but is incorporated in the proposed merge.
* Guidance to use a method if any of the stated prescriptions cannot be satisfied is currently in all three sections and remains in the proposed merge (as the first content after the list of them).
* An observation that `offsetTop` violates the second point and should have been a method is currently in all three sections and remains in the proposed merge (in its clearest form and with outbound links).

Reply:
@cynthia what do you think?